src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/TimeOutLicencesServiceTest.kt

```diff
@@ -0,0 +1,247 @@
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceEventType
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
+import java.time.Clock
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.ZoneId
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.StandardCondition as EntityStandardCondition
+
+class TimeOutLicencesServiceTest {
+  private val licenceRepository = mock<LicenceRepository>()
+  private val releaseDateService = mock<ReleaseDateService>()
+  private val auditEventRepository = mock<AuditEventRepository>()
+  private val licenceEventRepository = mock<LicenceEventRepository>()
+
+  private val service = Mockito.spy(
+    TimeOutLicencesService(
+      licenceRepository,
+      releaseDateService,
+      auditEventRepository,
+      licenceEventRepository,
+      clock,
+    ),
+  )
+
+  @BeforeEach
+  fun reset() {
+    val authentication = mock<Authentication>()
+    val securityContext = mock<SecurityContext>()
+
+    whenever(authentication.name).thenReturn("smills")
+    whenever(securityContext.authentication).thenReturn(authentication)
+    SecurityContextHolder.setContext(securityContext)
+
+    reset(
+      licenceRepository,
+      releaseDateService,
+      auditEventRepository,
+      licenceEventRepository,
+    )
+  }
+
+  @Test
+  fun `return if job execution date is on bank holiday or weekend`() {
+    whenever(releaseDateService.excludeBankHolidaysAndWeekends(LocalDate.now(clock))).thenReturn(true)
+
+    service.timeOutLicencesJob()
+
+    val cutOffDateCaptor = argumentCaptor<LocalDate>()
+
+    verify(releaseDateService, times(0)).getCutOffDateForLicenceTimeOut(cutOffDateCaptor.capture())
+    verify(licenceRepository, times(0)).getAllLicencesToTimeOut(cutOffDateCaptor.capture())
+  }
+
+  @Test
+  fun `should not update licences status if there are no eligible licences`() {
+    whenever(releaseDateService.excludeBankHolidaysAndWeekends(LocalDate.now(clock))).thenReturn(false)
+    whenever(releaseDateService.getCutOffDateForLicenceTimeOut(LocalDate.now(clock))).thenReturn(LocalDate.parse("2023-12-07"))
+    whenever(licenceRepository.getAllLicencesToTimeOut(LocalDate.now(clock))).thenReturn(emptyList())
+
+    service.timeOutLicencesJob()
+
+    val jobExecutionDateCaptor = argumentCaptor<LocalDate>()
+    val cutOffDateCaptor = argumentCaptor<LocalDate>()
+    val auditCaptor = ArgumentCaptor.forClass(AuditEvent::class.java)
+    val eventCaptor = ArgumentCaptor.forClass(LicenceEvent::class.java)
+
+    verify(releaseDateService, times(1)).getCutOffDateForLicenceTimeOut(jobExecutionDateCaptor.capture())
+    verify(licenceRepository, times(1)).getAllLicencesToTimeOut(cutOffDateCaptor.capture())
+
+    assertThat(jobExecutionDateCaptor.firstValue).isEqualTo(LocalDate.parse("2023-12-05"))
```

Comment:
rather than use captor you can just inline the value: 
```
    verify(releaseDateService, times(1)).getCutOffDateForLicenceTimeOut(LocalDate.parse("2023-12-05"))
```
The verify performs equals on the parameter to see if it has been called as expected